packages/api-v2/package.json

```diff
@@ -21,12 +21,11 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
-    "@graduate/common": "*",
+    "@graduate/common": "workspace:*",
```

Comment:
we should probably switch all the workspace imports to using `workspace:*`

Reply:
ahh yes!